src/modules/elements/action-button/style.pcss

```
@@ -4,14 +4,14 @@
 	/* extra class to override gutenberg styles */
 	&.tribe-editor__button,
 	&.tribe-editor__link {
-		display: flex;
 		align-items: center;
-		font-family: 'Helvetica Neue', Helvetica, -apple-system, BlinkMacSystemFont, Roboto, Arial, sans-serif;
-		font-size: 15px;
-		line-height: 18px;
-		letter-spacing: 0.38px;
-		color: #545D66;
-		text-decoration: none;
+		color: #727272 !important;
```

This color changed - and we added _yet another grey_ to our palette?
Can we use one that is already part of our palette?

---

`--tec-color-icon-primary-alt` is `#757575`. It's the closest
```suggestion
		color: var(--tec-color-icon-primary-alt) !important;
```

---

Thanks, applied the closest color as @codingmusician said